File: src/Task.java
public class Task {

    private String name;
    private boolean complete;

    public Task(String name, boolean complete) {
        this.name = name;
        this.complete = complete;
    }

    public String getName() {
        return name;
    }

    public boolean isComplete() {
        return complete;
    }

    public void complete() {
        complete = true;
    }

    public void incomplete() {
        complete = false;
    }
}

File: lib/edu/todo/task.rb
class Task
  attr_accessor :name, :desc, :is_complete, :due_date, :due_time

  def initialize(name)
    @name = name
    @is_complete = false
  end

  def hashit
    {
      "name" => @name,
      "desc" => @desc,
      "is_complete" => @is_complete,
      "due_date" => @due_date,
      "due_time" => @due_time
    }
  end
end
